src/sider/oppfolgingsplan/oppfolgingsplaner/OppfolgingsplanerOversikt.tsx

```diff
@@ -89,9 +112,28 @@ export default function OppfolgingsplanerOversikt(props: Props) {
   const hasNoInactivePlans =
     inaktivePlaner.length === 0 && oppfolgingsplanerLPSProcessed.length === 0;
 
+  const activeNarmesteLedere = !!latestOppfolgingstilfelle
+    ? activeNarmesteLederForCurrentOppfolgingstilfelle(
+        currentLedere,
+        latestOppfolgingstilfelle
+      )
+    : [];
+  const activeNarmesteLederIfSingle =
+    activeNarmesteLedere.length === 1 ? activeNarmesteLedere[0] : undefined;
+
   return (
     <div>
       <Sidetopp tittel="Oppfølgingsplaner" />
+      {!!aktivVeileder &&
+        !!activeNarmesteLederIfSingle &&
+        !!latestOppfolgingstilfelle && (
```

Comment:
Om vi ikke har et siste oppfølgingstilfelle gir det vel heller ikke mening å kunne be om oppfølgingsplan? 🤔

Reply:
Nei, for det er vel på bakgrunn av det man også finner nærmeste leder?
Burde vi også sjekke `hasActiveOppfolgingstilfelle` fra `useOppfolgingstilfellePersonQuery`() ? Ref https://nav-it.slack.com/archives/C08502GDMEV/p1734525609107679
Selv om man har latestOppfolgingstilfelle kan jo det være gammelt (og igjen utdatert nærmeste leder) :thinking: